client/components/webcamModule.js

```diff
@@ -0,0 +1,114 @@
+angular.module('in-your-face')
+.controller('webcamModuleCtrl', function($scope) {
+  console.log("123");
+  var _video = null;
+  var patData = null;
+  var self = this;
+
+  this.patOpts = {x: 0, y: 0, w: 25, h: 25};
+
+  // Setup a channel to receive a video property
+  // with a reference to the video element
+  // See the HTML binding in main.html
+  this.channel = {};
+  //
+  // this.webcamError = false;
+  // this.onError = function (err) {
+  //     $scope.$apply(
+  //         function() {
+  //             self.webcamError = err;
+  //         }
+  //     );
+  // };
+  //
+  this.onSuccess = function () {
+          // The video element contains the captured camera data
+    _video = this.channel.video;
+    $scope.$apply(function() {
+      _video.width = 540;
```

Comment:
_video.width = 540;
 _video.height =300;

This is hardcoded size assignment on canvas, may not work fine on different web cameras to resolution.
Images captured may be resized to fit the above hardcoded resolution.
Can we get it dynamically from this.channel.video.width or this.channel.video.height?